hathor/transaction/storage/transaction_storage.py

```diff
@@ -45,6 +53,9 @@ def __init__(self):
         # Cache for the best block tips
         self._best_block_tips = []
 
+        # Cache for the latest timestamp of all tips with merkle tree precalculated to be used on the sync algorithm
```

Comment:
I think a better explanation would be good. As far as I understand, this cache is always kept up to date with `self._latest_timestamp` and is invalidated all time a new txs arrives.

Reply:
Added an explanation about the cache being invalidated on the add cache and about the latest_timestamp.

Can you see if it's good?